packages/core-api/src/apis/definitions/ScmIntegrationsApi.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ScmIntegrationRegistry } from '@backstage/integration';
```

Comment:
With this we need to remember that breaking changes in `@backstage/integrations` might cause breaking changes in core as well. Tbh I think it might actually be worth redefining the interface here instead, and omitting the pieces that we don't want to include in the frontend, like secrets in config. Would also be an opportunity to be a bit more experimental in the integrations API while listing out more stable bits here.

Reply:
Hm. It's a nontrivial surface. I guess I could do that but it'll be a bit gnarlier

Reply:
Since this got moved to a dedicated package instead, the versioning dependency has been resolved